examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4770</td>\n",
+       "      <td>male</td>\n",
+       "      <td>1999-07-01</td>\n",
+       "      <td>[\"230 Olson Fort Suite 50\"]</td>\n",
+       "      <td>[\"Dennis\"]</td>\n",
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4776</td>\n",
+       "      <td>female</td>\n",
+       "      <td>1993-07-31</td>\n",
+       "      <td>[\"612 Kuhlman Skyway Apt 45\"]</td>\n",
+       "      <td>[\"Amherst\"]</td>\n",
+       "      <td>[\"MA\"]</td>\n",
+       "      <td>[]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     id  gender  birthdate                         street             city  \\\n",
+       "0  4765    male 1984-09-09    [\"754 Feil Tunnel Unit 36\"]  [\"Springfield\"]   \n",
+       "1  4767  female 1999-04-26  [\"766 King Landing Suite 14\"]      [\"Spencer\"]   \n",
+       "2  4768  female 1999-02-12             [\"508 Auer Lodge\"]    [\"Wilbraham\"]   \n",
+       "3  4770    male 1999-07-01    [\"230 Olson Fort Suite 50\"]       [\"Dennis\"]   \n",
+       "4  4776  female 1993-07-31  [\"612 Kuhlman Skyway Apt 45\"]      [\"Amherst\"]   \n",
+       "\n",
+       "    state        zip  \n",
+       "0  [\"MA\"]  [\"01119\"]  \n",
+       "1  [\"MA\"]  [\"01562\"]  \n",
+       "2  [\"MA\"]         []  \n",
+       "3  [\"MA\"]         []  \n",
+       "4  [\"MA\"]         []  "
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# For this dataset we interpret the current address as one where period is empty.\n",
+    "\n",
+    "current = pats.address.where(pats.address.period.empty())\n",
+    "\n",
+    "simple_pats = pats.select(\n",
+    "    {\n",
+    "        \"id\": pats.id,\n",
+    "        \"gender\": pats.gender,\n",
+    "        \"birthdate\": pats.birthDate,\n",
+    "        \"street\": current.line,\n",
+    "        \"city\": current.city,\n",
+    "        \"state\": current.state,\n",
+    "        \"zip\": current.postalCode,\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "runner.to_dataframe(simple_pats, limit=5)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "That's nice, but suppose we want to create it as an actual Spark view -- basically a virtual table that can be easily used by any application that uses Spark. We can simply turn that definition into a Spark view"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "runner.create_database_view(simple_pats, \"patients\")"
```

Comment:
nit: Let's use a more descriptive name than just `patients`, e.g., maybe `patient_current_address` or a shorter version.

Reply:
Any comments on this? I think this is not resolved, is it?